### ISSUE TITLE
Add explanatory tooltips to registration screens

### ISSUE THREAD

**Issue by jindrichmynarz** [opendatacz/pc-filing-app#43]
Registration screen can benefits from additional tooltips describing the individual form fields. For example, what is "IC (optional)"?

**Comment by jindrichmynarz** [comment]
@jakhrkal, please add help messages:

``` bash
cd src/main/resources/cz/opendata/tenderstats/i18n
grep TODO `find register*`
```

**Comment by jakhrkal** [comment]
Should be solved by 36aebae7ccd76eb4b46b872fbc24a364bbc9abf9 on develop. Please check and if so, close the issue.
